docs/modelserving/storage/s3/mnist-s3.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: "serving.kserve.io/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: "mnist-s3"
+spec:
+  predictor:
+    serviceAccountName: sa
+    tensorflow:
+      storageUri: "s3://kserve-examples/mnist"
```

Comment:
Does this s3 bucket exist?

Reply:
@yuzisun we have this s3 bucket but it is private.